[Swift 6]: Update Expert mixolgist and its concept (#833)
exercism/swift: concepts/control-transfer/introduction.md

# Introduction

Swift has a set of statements that can be used to alter the normal control flow of loops.
These are known as control transfer statements.

> And you have already seen one of them, `return`, but this concept will introduce the other onces, which iclude `continue`, `break`, and labels.
> And one which we wont cover in this concept, `fallthrough`, but you can read about it in the [Swift book][fallthrough].

Reviewer: ```suggestion
You have already seen `return`; this concept will introduce other, which include `continue`, `break`, and labels.
We won't cover `fallthrough` in this concept, but you can read about it in the [Swift book][fallthrough].
```

> ## continue

Reviewer: ```suggestion
## Continue
```

Sometimes it is necessary to skip to the next iteration of a loop early, without completing the rest of the statements in the current iteration of the loop.
The `continue` keyword can be used for this.

> When `continue` is executed, the loop jump to the next check to see if the next iteration of the loop can be run, i.e. the `while` in while and repeat-while loops or the check if there's another element in the sequence in for-in loops.

Reviewer: ```suggestion
When `continue` is executed, the loop jumps to the next check to see if the next iteration of the loop can be run, i.e. the `while` in while and repeat-while loops or the check if there's another element in the sequence in for-in loops.
```
